examples/with-google-analytics/pages/_app.js

```diff
@@ -17,27 +18,31 @@ const App = ({ Component, pageProps }) => {
 
   return (
     <>
+    <Head>
       {/* Global Site Tag (gtag.js) - Google Analytics */}
-      <Script
-        strategy="afterInteractive"
-        src={`https://www.googletagmanager.com/gtag/js?id=${gtag.GA_TRACKING_ID}`}
-      />
-      <Script
-        strategy="afterInteractive"
+      <script
         dangerouslySetInnerHTML={{
           __html: `
             window.dataLayer = window.dataLayer || [];
             function gtag(){dataLayer.push(arguments);}
             gtag('js', new Date());
 
+            // Disable any default-enabled advertising features (turn them on later when we get consent)
+            gtag('set', 'allow_google_signals', false);
```

Comment:
IMHO we shouldn't add this in the PR:

- It is not required to set up Google Analytics
- Anyone who follows Google Analytics' instruction can add this easily

Reply:
This is not part of the PR anymore. ill update the description, sorry for that

Reply:
https://github.com/vercel/next.js/pull/43894/files

Reply:
This is the commit that removes this code 4 days ago

https://github.com/vercel/next.js/pull/43894/commits/581c0af2ae06c1200ec055a7196394e9ea89d65e